extensions/firefox/content/PdfJs.jsm

```diff
@@ -173,7 +173,7 @@ var PdfJs = {
 
   updateRegistration: function updateRegistration() {
     if (this.enabled) {
-      this._ensureRegistered();
+      this.ensureRegistered();
```

Comment:
That sounds reasonable. I just left it private because I didn't actually need it, but it is a little weird to make it different.